app/models/user.rb

```diff
@@ -9,33 +9,48 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+  has_many :sent_requests, class_name: 'Friendship', foreign_key: 'user_id', dependent: :destroy
+  has_many :received_requests, class_name: 'Friendship', foreign_key: 'friend_id', dependent: :destroy
 
-  has_many :friendships
-  has_many :inverse_friendships, class_name: 'Friendship', foreign_key: 'friend_id'
-
-  def friends
-    friends_array = friendships.map { |friendship| friendship.friend if friendship.confirmed }
-    friends_array + inverse_friendships.map { |friendship| friendship.user if friendship.confirmed }
-    friends_array.compact
+  def accepted_friendships
+    sent_requests.where(confirmed: true)
```

Comment:
- [ ] [OPTIONAL] `User#confirmed_friendships` can be defined as association with SQL query like below

```ruby
has_many :confirmed_friendships, -> { where confirmed: true }, class_name: "Friendship"
has_many :friends, through: :confirmed_friendships
```